lib/datadog/ci/recorder.rb

```diff
@@ -48,23 +47,22 @@ def trace(span_type, span_name, tags: {}, &block)
 
       def create_datadog_span(span_name, span_options: {}, tags: {}, &block)
         if block
-          ::Datadog::Tracing.trace(span_name, **span_options) do |tracer_span, trace|
+          Datadog::Tracing.trace(span_name, **span_options) do |tracer_span, trace|
```

Comment:
Did I googled correctly if my understanding is that this "moves" resolution for `Datadog` from global to local?

Did the analytics class need to be global before, and since you're not using it anymore in this context, it's now safe to use the imported reference? Or was it already unnecessary to begin with?

Reply:
it was unnecessary from the beginning and tbh I am just bad with this kind of stuff and very often use `::` prefix where I don't have to